saleor/product/migrations/0172_create_default_category_and_product_type.py

```diff
@@ -0,0 +1,42 @@
+from django.db import migrations
+from django.conf import settings
+
+from ...product import ProductTypeKind
+
+
+def create_default_category(apps, schema_editor):
+    Category = apps.get_model("product", "Category")
+    if not Category.objects.all().exists() and not settings.BLOCK_POPULATE_DEFAULTS:
+        Category.objects.create(
+            name="Default Category",
+            slug="default-category",
+            lft=0,
+            rght=0,
+            tree_id=0,
+            level=0,
+        )
+
+
+def create_default_product_type(apps, schema_editor):
+    ProductType = apps.get_model("product", "ProductType")
+    if not ProductType.objects.all().exists() and not settings.BLOCK_POPULATE_DEFAULTS:
+        ProductType.objects.create(
+            name="Default Type",
+            slug="default-type",
+            kind=ProductTypeKind.NORMAL,
+            has_variants=False,
```

Comment:
Is it expected that we create Simple ProductType?

Reply:
It's been discussed already; see my comments above.